### ISSUE TITLE
A11y_mssql for VSCode_Results Tab_Non-text Content: Tooltips are not available for all the non-text content in the results tab for keyboard only users

### ISSUE THREAD

**Issue by Tanushagottipati** [microsoft/vscode-mssql#1231]
**Environment Details:**
Visual Studio Code – Insider (install from https://na01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fcode.visualstudio.com%2Finsiders.&data=02%7C01%7C%7C00d9f7de52714b323f9908d607afe0c5%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C636704849297354687&sdata=cmMqckjoNPGtCS3SZ00bCqCnFprKgiux%2BIjyvzgU0hE%3D&reserved=0)
Version: 1.34.0-insider (user Setup)
Mssql version: 1.6.0
OS Version: Microsoft Windows 10 Enterprise (10.0.18362 Build 18362)
Screen reader: NVDA, JAWS

**MAS Violated:** MAS 1.1.1

**Repro Steps:**
1.      Open Visual Studio Code in your system
2.      Navigate to "Extensions"   and install "SQl Server (mssql)" extension
3.      Press F1 and select MS SqL :   Connect and connect to any sql server
4.      Write code (SELECT * FROM sys.objects;) in the new document and press F1 and select ""MS SQL : Execute Query""    
5.       Navigate to all the controls in the page   
6.       Verify if tooltips are present for all the non-text content present in the page

**Actual Result:** 
Tooltips are not available for all the non-text content in the results tab for keyboard users

**Expected Result:** 
Tooltip must be visible for all the non-text content in the results tab (More options, save to CSV,JSON, Excel, Split Window and for the data in the table which is not visible completely)

**User Impact:** 
Keyboard only users will find it difficult in understanding the non-text content if proper tooltips are not provided 

**MAS Reference:** 
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={d2d2051f-bdc8-4af7-8e18-38aae867e216}

![Bug_1231_Tooltips are not available for all the non-text content in the results tab for keyboard users](https://user-images.githubusercontent.com/41109135/56642142-707ddb00-6694-11e9-9dd7-164a3de82189.png)

**Comment by geleems** [comment]
I do not understand exactly the meaning by "Tooltips are not available for all the non-text content in the results tab". What is the expected behavior exactly?
1) Did you mean tooltip should be shown when a cell of result table is focused with keyboard?
2) Did you mean tooltip should be shown when a icon (such as 'Save to CSV') is focused by keyboard?
3) What is current unique behavior on 'non-text' content over 'text' content in the result table?
4) Why did you draw redbox on those icons in the screenshot? What do they wrong?

**Comment by Tanushagottipati** [comment]
@geleems 
Here there are two issues. Sorry for putting both the issue in one screenshot. let me Update it.
1. First issue is for the Non-text content (Icons) are not visible to the users when they are focused with Keyboard
2. Second issue is, for the text present in the Results table which is not visible completely while navigating with the keyboard. So if a tooltip is provided on keyboard focus then Keyboard users can know the complete text

**Comment by geleems** [comment]
@Tanushagottipati 
Thank you very much for the clarification.

**Comment by alanrenmsft** [comment]
@Tanushagottipati VSCode also have the same issue and it is being tracked by this work item: https://github.com/Microsoft/vscode/issues/52016, I had an offline conversation with Ryan Shugart, and he suggests to close this issue since we have an external dependency.

**Comment by Tanushagottipati** [comment]
Based on the above comments closing this bug.

**Comment by fsteffi** [comment]
#Closed;
